content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-5
+Category: Events
+Tags: social, dropbox
```

Comment:
I'd rather not have a dropbox-specific tag---maybe we could switch this to our general "industry" one?